src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -41,12 +41,8 @@ class AccessDecisionManager implements AccessDecisionManagerInterface
      *
      * @throws \InvalidArgumentException
      */
-    public function __construct(array $voters, $strategy = self::STRATEGY_AFFIRMATIVE, $allowIfAllAbstainDecisions = false, $allowIfEqualGrantedDeniedDecisions = true)
+    public function __construct(array $voters = array(), $strategy = self::STRATEGY_AFFIRMATIVE, $allowIfAllAbstainDecisions = false, $allowIfEqualGrantedDeniedDecisions = true)
```

Comment:
This is not required, but if setting no voters on instantiation is allowed,  I thought it would be more consistent to make the argument optional also.